server/routes/admin/articles/articles.js

```diff
@@ -29,16 +29,32 @@ router.post('/add', (req, res) => {
     articleClient.addArticle(query, callBack)
 });
 
+
+
+
+
+
 router.get('/edit/:articleId', (req, res) => {
     const {articleId} = req.params;
-    const callback = (error, article) => {
-        res.render("admin-edit-article", {
-            article: article
-        })
+    const callback = (error, categories) => {
+        callback2 = (error, article) => {
```

Comment:
don't call them callback1 and callback2 - one is `categoriesCallback`, the other is `articleCallback`